python/seldon_core/flask_utils.py

```diff
@@ -1,10 +1,12 @@
-import json
 import base64
-
+import json
 from http import HTTPStatus
-from flask import request, current_app, jsonify as flask_jsonify
 from typing import Dict, Union
 
+from flask import current_app
+from flask import jsonify as flask_jsonify
+from flask import request
```

Comment:
Does this need to be split into separate lines? 
If yes, I'd prefer to use `( ... )` like we have in other places.

Reply:
I see. it's due to the aliasing of `jsonify as flask_jsonify`. Otherwise isort would put them into one line like
```
from flask import current_app, jsonify, request
```

Reply:
Oh, I didn't realise we had aliasing in one line :D
I guess the best is to have then
```
from flask import current_app, request
from flask import jsonify as flask_jsonify
```
?

Reply:
it's because `request` comes after `jsonify` alphabetically. if it's like

```
from flask import ctx, current_app
from flask import jsonify as flask_jsonify
```
isort would keep `ctx` and `current_app` in one line.

Reply:
What if we use `# isort:skip`:
```
from flask import current_app, request
from flask import jsonify as flask_jsonify   # isort:skip
```
?

Reply:
I think the main benefit of isort is that it has a set of rules for ordering imports and consistent. Perhaps we could customize then, but then it feels ad-hoc just like the current heuristic way unless we have a good reason like to be compatible with black.

I see it this way, like black, it is opinioned but consistent.

Reply:
tried, I didn't know this, but isort would put a line in between
```
from flask import current_app, request

from flask import jsonify as flask_jsonify   # isort:skip
```

Reply:
Ok, cool, let's leave it like that

Reply:
And by that I meant with ` # isort:skip` ; - )